p2p/connection_test.go

```diff
@@ -110,6 +113,37 @@ func TestMConnectionStatus(t *testing.T) {
 	assert.Zero(status.Channels[0].SendQueueSize)
 }
 
+func TestPingPongTimeout(t *testing.T) {
```

Comment:
So this is supposed to test that we throw an error if we don't receive a pong? I think it needs some modifications to ensure it doesn't block on `flush`. Also, can we test that we don't error if we do send the pong ?